### DISCUSSION TITLE
hey wanted to ask on your branch needs to change from 0.5 to 0.8 for better inpaint faceswap? ty

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [mashb1t/Fooocus#22] [Q&A]
hey wanted to ask on your branch needs to change from 0.5 to 0.8 for better inpaint faceswap? ty
also was trying to find the line didnt see it ty so much ( main branch)

**Comment by mashb1t** [comment]
I'll merge Fooocus back to my fork in a few days.
